lib/maths/CBoostedTreeImpl.cc

```diff
@@ -444,6 +445,11 @@ void CBoostedTreeImpl::train(core::CDataFrame& frame,
         LOG_TRACE(<< "Test loss = " << m_BestForestTestLoss);
 
     } else if (m_CurrentRound < m_NumberRounds || m_BestForest.empty()) {
+        using TUIntMeanVarAccumulator =
+            CBasicStatistics::SSampleMeanVar<std::uint64_t>::TAccumulator;
```

Comment:
This doesn't like integers. We could do something to manage integer division better, but it is probably better to stick to double and cast when adding the iteration time.